src/docs/components/tabs.hbs

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  {{> html-head title="Tabs" }}
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <h2>Tabs Product</h2>
+  </div>
+  <ul class="nav nav-tabs mx-auto nav-tabs-product text-center mb-5" role="tablist">
```

Comment:
please add also fake contents for each tab, just a different Lorem Ipsum for each